tests/conftest.py

```diff
@@ -71,3 +71,43 @@ def tls_ca_ssl_context(tls_certificate_authority: trustme.CA) -> ssl.SSLContext:
     ssl_ctx = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
     tls_certificate_authority.configure_trust(ssl_ctx)
     return ssl_ctx
+
+
+@pytest.fixture(scope="package")
+def reload_directory_structure(tmp_path_factory: pytest.TempPathFactory):
+    """This fixture creates a directory structure to enable reload parameter tests"""
```

Comment:
```suggestion
    """
    This fixture creates a directory structure to enable reload parameter tests
    
    ├── app
    │ ├── css
    │ │ └── main.css
    │ ├── js
    │ │ └── main.js
    │ ├── src
    │ │ └── main.py
    │ └── sub
    │     └── sub.py
    ├── app_first
    │ ├── css
    │ │ └── main.css
    │ ├── js
    │ │ └── main.js
    │ ├── src
    │ │ └── main.py
    │ └── sub
    │     └── sub.py
    ├── app_second
    │ ├── css
    │ │ └── main.css
    │ ├── js
    │ │ └── main.js
    │ ├── src
    │ │ └── main.py
    │ └── sub
    │     └── sub.py
    ├── app_third
    │ ├── css
    │ │ └── main.css
    │ ├── js
    │ │ └── main.js
    │ ├── src
    │ │ └── main.py
    │ └── sub
    │     └── sub.py
    ├── ext
    │ └── ext.jpg
    └── main.py
    """
```

Reply:
not sure about that suggestion, just something I may want to see because reading the test it was not obvious from me reading the fixture what the directory layout looked like, but this is very long so....

Reply:
I've taken the middle road and just shortened the directories down to one example with indication they are all the same